fitapp/tasks.py

```diff
@@ -25,8 +26,7 @@ def subscribe(fitbit_user, subscriber_id):
         fb = utils.create_fitbit(**fbuser.get_user_data())
         try:
             fb.subscription(fbuser.user.id, subscriber_id)
-        except:
-            exc = sys.exc_info()[1]
+        except Exception as exc:
             logger.exception("Error subscribing user: %s" % exc)
```

Comment:
@brad Shoot, I wonder how many exceptions we have been swallowing here without logging them. Thanks for fixing these.